### PR TITLE
Create a new tutorial for JMS SQS use case

### PR DESCRIPTION
**Pull request by sanjaypujare**
This is a new example for using the new JMS Input operator for the SQS use-case. @amberarrow  could you review?

### CODE REVIEW

**Comment by amberarrow** [comment]
Suggest prepending "SPOI-8769 " to commit message

**Comment by amberarrow** [comment]
Some of my comments on the ActiveMQ example, _mutatis mutandis_, also apply here.

File: tutorials/jmsSqs/src/main/java/com/example/jmsSqs/SqsApplication.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  {
+	  
+    JMSStringInputOperator sqsInput = dag.addOperator("sqsIn", 
+              new JMSStringInputOperator());
+    
+    MyConnectionFactoryBuilder factoryBuilder = new MyConnectionFactoryBuilder();
+    
+    factoryBuilder.sqsDevCredsFilename = conf.get(SQSDEV_CREDS_FILENAME_PROPERTY);
+    
+    sqsInput.setConnectionFactoryBuilder(factoryBuilder);
```
Comment (amberarrow) [inline tutorials/jmsSqs/src/main/java/com/example/jmsSqs/SqsApplication.java]: Can some/all of this be done in the setup() method of the operator with the filename injected from the properties file ?

Comment (sanjaypujare) [inline tutorials/jmsSqs/src/main/java/com/example/jmsSqs/SqsApplication.java]: @amberarrow  the factoryBuilder cannot be set from the properties file (for obvious reasons) but the other properties I might be able too. I'll look into that.

Comment (sanjaypujare) [inline tutorials/jmsSqs/src/main/java/com/example/jmsSqs/SqsApplication.java]: @amberarrow  In the latest commit I have made the changes as you suggested - setting the values thru properties.

Comment (amberarrow) [inline tutorials/jmsSqs/src/main/java/com/example/jmsSqs/SqsApplication.java]: Ok, I'll take a look.

Comment (sanjaypujare) [inline tutorials/jmsSqs/src/main/java/com/example/jmsSqs/SqsApplication.java]: Pls wait until I finish all the changes....

**Comment by amberarrow** [comment]
@sanjaypujare Not sure if all the changes are done; I'll wait till you confirm before I review again.

**Comment by sanjaypujare** [comment]
PR https://github.com/DataTorrent/examples/pull/34 replaces this PR. Closing this one...

**Pull request closed without merging**